pkg/cli/admin/upgrade/status/status.go

```diff
@@ -282,8 +282,9 @@ func (o *options) Run(ctx context.Context) error {
 	var updateInsights []updateInsight
 	var workerPoolsStatusData []poolDisplayData
 	var controlPlanePoolStatusData poolDisplayData
+	multiArchMigrationInProgress := multiArchMigration(cv.Status.History) && progressing.Status == configv1.ConditionTrue
```

Comment:
> Acknowledge limitations and be ok with them.

I prefer to this one too.
I admit that data plane update is ignored for the technical difficulty.
Arguable, it might be an even better behaviour. See my comment.

https://github.com/openshift/oc/pull/1933/files#diff-a7caf221970b9d8c0d7e98b344e66c2f5d88490c787d696f8162b2d62630bc05R285-R290

"Ideally MCO do not have to reboot the existing machines for the machine-config changes caused by the multi-arch migration" is discussed https://redhat-internal.slack.com/archives/C02CZNQHGN8/p1733410304411049